Web_Development/Javascript/Topics/5.Syntax/2.Operators/README.md

```diff
@@ -0,0 +1,298 @@
+# To Do
+#JavaScript Operators
+
+**Operators** are used to perform some action on operands. Operand can be of any data type, eg. number, boolean, string.
+
+Some of the operators are described be -
+
+## **1. Arithmetic Operators**
+Arithmetic operators take numerical values as operands and evaluates it to a single numerical value. Some of the arithmetic operators are -
+
+
+* **Addition(+)** - It adds the numerical operands and is also used for string concatenation.It will add numerical and string operand to a number if the string can be converted to a number. Else it concatenates them.
+
+    **Example:**  
+    ```js
+                  var x = 10;      // assign the value 10 to x
+                  var y = 4;       // assign the value 4 to y
+                   alert(x + y);    // 0utput: 14
+    ```                     
```

Comment:
You don't need to indent the code-blocks!

````
    ```js
    var x = 10;      // assign the value 10 to x
    var y = 4;       // assign the value 4 to y
    alert(x + y);    // 0utput: 14
    ```                     
````